it/test/uk/gov/hmrc/tradergoodsprofilesrouter/MaintainProfileIntegrationSpec.scala

```diff
@@ -45,19 +48,24 @@ class MaintainProfileIntegrationSpec extends HawkIntegrationSpec with AuthTestSu
     when(dateTimeService.timestamp).thenReturn(Instant.parse("2021-12-17T09:30:47.456Z"))
   }
 
-  override def extraApplicationConfig: Map[String, Any] = {
+  override def extraApplicationConfig: Map[String, Any] =
     super.extraApplicationConfig ++ Map(
       "features.isDrop1_1_enabled" -> false
     )
-  }
 
-  val headers: Seq[(String, String)] =
+  // TODO: After drop 1.1 remove x-client-id from headers - Ticket: TGP-2014
+  val headers: Seq[(String, String)] = if (appConfig.isClientIdHeaderDisabled) {
```

Comment:
This as well it does not need the if statement as the isClientIdHeaderDisabled is always false

Reply:
Removed the if statement for the headers in the test